### PR TITLE
fix: navigate to my collection after successful purchase

### PR DESCRIPTION
**Pull request by rajsam003**
The type of this PR is: Bugfix

### Description

Paired with @MounirDhahri 

https://user-images.githubusercontent.com/11945712/144619761-187c157e-69c5-4956-9c60-f8af15c71a6e.mov

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [ x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- navigate to my collection after purchase - mounir, sam

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Fix build failure due to widget - mounir, sam

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2f88b1a7f8e2982908f777a8d2f6280f49dd6f26
</p>

File: ArtsyWidget/FeaturedArtworks/FeaturedArtworks+MediumView.swift [hunk 1]
```diff
@@ -85,14 +85,14 @@ extension FeaturedArtworks {
             GeometryReader { geo in
                 VStack(spacing: 0) {
                     TopArtwork(artwork: primaryArtwork)
-                        .frame(height: geo.size.height * topFactor)
+                        .frame(height: geo.size.height * CGFloat(topFactor))
```
Comment (lordkiz) [inline ArtsyWidget/FeaturedArtworks/FeaturedArtworks+MediumView.swift:88]: Cool

Comment (MounirDhahri) [reply ArtsyWidget/FeaturedArtworks/FeaturedArtworks+MediumView.swift:88]: this was causing the build to fail on master

**Review by lordkiz — APPROVED** [review]
This looks good to me.

**Review by brainbicycle — DISMISSED** [review]

File: ArtsyWidget/FeaturedArtworks/FeaturedArtworks+MediumView.swift (hunk 1 above)
Comment (jonallured) [reply ArtsyWidget/FeaturedArtworks/FeaturedArtworks+MediumView.swift:88]: Whoaaa, I wonder why!

**Pull request merged**